generate_chialisp_constants.py

```diff
@@ -347,6 +465,8 @@
 os.makedirs(os.path.dirname(rust_dest_path), exist_ok=True)
 os.makedirs(os.path.dirname(python_dest_path), exist_ok=True)
 
+error_count = 0
```

Comment:
You can resolve this comment but for what it's worth, this change is unrelated to adding the new puzzles and should be its own PR